src/components/Quote.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Calculator from './Calculator';
+
+const Quote = ({ category }) => {
+  const [quote, setQuote] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      try {
+        const url = `https://api.api-ninjas.com/v1/quotes?category=${category}`;
+        const response = await fetch(url, {
+          headers: {
+            'X-Api-Key': '647a531e06f737362c987227',
```

Comment:
- [ ] Your API key is still returning invalid. Please ensure that your API key is generated correctly. Also, ensure you test it and be sure it's working.

![image](https://github.com/theresetuyi/Math-magician/assets/63473584/73abefa5-4cce-4a59-8d28-e4bdf6ab8708)